lib/graphql/schema/enum.rb

```diff
@@ -223,6 +227,20 @@ def inherited(child_class)
         def own_values
           @own_values ||= {}
         end
+
+        def generate_value_method(value, configured_value_method)
+          return if configured_value_method == false
+
+          value_method_name = configured_value_method || value.graphql_name.downcase
+
+          if respond_to?(value_method_name.to_sym)
+            $stderr << "Failed to define value method for :#{value_method_name}, because " \
+              "#{value.owner.name} already responds to that method. Use `value_name:` to override the method name."
```

Comment:
```suggestion
            warn "Failed to define value method for :#{value_method_name}, because " \
              "#{value.owner.name} already responds to that method. Use `value_name:` to override the method name."
```